src/Views/NowPlayingView.vala

```diff
@@ -25,17 +26,16 @@ public class Music.NowPlayingView : Gtk.Grid {
 
         var seekbar = new Music.SeekBar ();
 
-        var play_pause_image = new Gtk.Image.from_icon_name (
-            "media-playback-start-symbolic",
-            Gtk.IconSize.LARGE_TOOLBAR
-        );
+        var play_pause_image = new Gtk.Image.from_icon_name ("media-playback-start-symbolic") {
+            pixel_size = 24
+        };
 
         var play_button = new Gtk.Button () {
             action_name = Application.ACTION_PREFIX + Application.ACTION_PLAY_PAUSE,
+            css_classes = {"flat"},
             halign = Gtk.Align.CENTER,
-            image = play_pause_image
+            child = play_pause_image
```

Comment:
GtkButton has `icon-name` now. :) But here: `new Gtk.Button.from_icon_name`. And this will also add `.image-button` style class, you will have to remove this `css_classes` as it breaks that.

Reply:
Yeah the problem with that is it doesn't seem there's a way to set icon size, so this seems the way for that unless we use CSS later